src/controllers/UserController.js

```diff
@@ -0,0 +1,43 @@
+const User = require('../models').User;
+const getHashedPassword = require('../helpers/helpers');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+module.exports = {
+  async register(req, res) {
+    try {
+      const { username, password } = req.body;
+      const alreadyExistsUser = await User.findOne({ where: { username } });
+      if (alreadyExistsUser) {
+        return res.status(500).send('User with this username already exists');
+      }
+      const hashedPassword = await getHashedPassword(password);
+      const newUser = await new User({ username, password: hashedPassword });
+      await newUser.save();
```

Comment:
```suggestion
      const newUser = await new User({ username, password: hashedPassword });
      await newUser.save();
```
надо держаться одной стили. Посмотри другие контроллеры (например создание/получение/редактирование записи)

Reply:
исправлено